app/templates/static/layout.static.pug

```diff
@@ -32,37 +32,38 @@ html(lang='en')
       script(src=chunkPaths[viewName], defer='')
     script(src='/' + shaTag + '/javascripts/app.js', defer='', async='', data-ace-base='/javascripts/ace') // This data tag needs to be on SOME script with a src for Ace to work
     script(src='/user-data?sha=' + shaTag)
-    // Google Analytics
-    script.
-      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
-      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
-      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
-      })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
-      ga('create', 'UA-39724129-1', 'auto');
+    if me.useGoogleAnalytics()
+      // Google Analytics
+      script.
+        (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+        (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+        m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+        })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
+        ga('create', 'UA-39724129-1', 'auto');
 
-      if(features.codePlay) { ga('create', 'UA-80597440-3', {name: 'codeplay'}); }
+        if(features.codePlay) { ga('create', 'UA-80597440-3', {name: 'codeplay'}); }
 
-    // Global site tag (gtag.js) - Google Ads: 811324643
-    script(async src="https://www.googletagmanager.com/gtag/js?id=AW-811324643")
-    script.
-      window.dataLayer = window.dataLayer || [];
-      function gtag(){dataLayer.push(arguments);}
-      gtag('js', new Date());
-      gtag('config', 'AW-811324643');
+      // Global site tag (gtag.js) - Google Ads: 811324643
```

Comment:
Do we need to apply the same logic to snowplow / Google Ads?  Are these allowed in China?  Can they run?

May not be appropriate changes for this PR but want to call out for discussion.

Reply:
The `if me.useGoogleAnalytics()` conditional should prevent all of these analytics tools from loading.